VastClient/VastClient/models/XML schema/VastExtension.swift

```diff
@@ -13,13 +13,12 @@ struct ExtensionAttributes {
 }
 
 struct ExtensionElements {
-    static let ext = "Extension" // extension is an invalid identifier
     static let creativeparameters = "CreativeParameters" // TODO: this needs to be defined outside the library
     static let creativeparameter = "CreativeParameter" // TODO: this needs to be defined outside the library
 }
 
 public struct VastExtension {
-    public var type: String
+    public let type: String
     public var creativeParameters = [VastCreativeParameter]()
```

Comment:
@jgainfort Can you provide me with example XML VAST file with Creative parameter being used so I could make sure we did not break your implementation?

Reply:
http://29773.v.fwmrm.net/ad/g/1?nw=169843&mode=live&resp=vmap1&prof=nbcu_playmaker_ios&csid=nbcsports_ctv_live_appletv&caid=nbcs_155574&asnw=169843&ssnw=169843&vdur=600&pvrn=2474489029553960897&vprn=3433621882520324122&flag=+qtcb-fbad-vicb+slcb+sltp+amsl-emcr-play-uapl&metr=1031&afid=140315314&sfid=7089953&vip=65.153.144.238&vclr=3.1.0;_fw_h_user_agent=NBCU/1.0%20(iPhone)&_fw_ae=725bd42fc9c9394e790b9a73610da431&_fw_vcid2=169843:B51D0835-F3E9-4A32-987A-394F45824713&ltlg=39.7393,-104.9844&_fw_player_width=375&_fw_player_height=220;slid=mid6&slau=midroll&tpos=30888.226000002305&ptgt=a&maxd=45.0&mind=45.0&cpsq=6